### ISSUE TITLE
When message & file both sent, Preview text not shown

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop-client#995]
## Description
When both a file and a message are received, the Preview text that shows is `<Encrypted file on server>`  instead of the first 2 lines of text from the message. 

Note, I have gotten this to reproduce multiple times from 2 sources w/ the SuperFun instance.

## STR
1. Submit file AND message, either as new source or as a follow-up submission from a source
2. Receive both, in the client.
3. Either click "yes" on the GPG dialog, or not (it's happened both in follow-up syncs and on the first one).
4. Watch client decrypt/show message.
5. `<Encrypted file on server>` message still shows
6. 15min later, send new message, but no file.
7. New message appears at the same time Preview snippet appears.

## Expected
When a message and file are both sent, the text of the sent message should show in the Preview area. The `<Encrypted file on server>` blab is only for when just a file has been sent.

![image](https://user-images.githubusercontent.com/8262612/77492981-226edb80-6dff-11ea-8bfb-136e68947397.png)

**Comment by eloquence** [comment]
I don't think this is a bug or a regression; the first iteration of previews that landed in the client simply never fully implemented the behavior described in #135. 

> When a message and file are both sent, the text of the sent message should show in the Preview area. The <Encrypted file on server> blab is only for when just a file has been sent.

As you pointed out in https://github.com/freedomofpress/securedrop-client/issues/135#issuecomment-488401357, this implies some kind of time window to group related activity, which adds significant complexity to the preview logic. Otherwise you can't detect this case:

1. Source sends a message
2. Source waits 2 days
3. Source sends a file

In that case, the journalist will want to know that a file has newly been sent by the same source, so we need to be careful not to "swallow" that information.

In spite of that added complexity, this may be the best solution. I agree that ending up with lots of `<Encrypted file on server>` preview snippets could render previews far less useful. But I think this is still an area where it's worth considering some alternative design solutions, with user research from the pilot as an important input.

**Comment by redshiftzero** [comment]
This behavior was unimplemented, putting on milestone for doing during the pilot

**Comment by zenmonkeykstop** [comment]
Still an active bug. Will address independently or as part of potential refactoring.

**Comment by eloquence** [comment]
Closing, non-critical issue specific to the current codebase/UX, which is undergoing a rewrite.
